www/build-themes.js

```diff
@@ -11,7 +11,13 @@ const importer = function importer(url) {
   if (url.startsWith('~paragon-style')) {
     file = path.resolve(__dirname, '../styles', url.substr('~paragon-style/'.length));
   } else if (url[0] === '~') {
-    file = path.resolve(__dirname, '../node_modules', url.substr(1));
+    const siblingNodeModulesPath = path.resolve(__dirname, './node_modules', url.substr(1));
+    const parentNodeModulesPath = path.resolve(__dirname, '../node_modules', url.substr(1));
+    if (fs.existsSync(siblingNodeModulesPath)) {
+      file = siblingNodeModulesPath;
+    } else {
+      file = parentNodeModulesPath;
+    }
```

Comment:
[infom] While doing local development, at one point NPM installed the `@edx/elm-theme` package in the `www/node_modules`, instead of getting hoisted to the root `node_modules`. This occurred when installed `@edx/elm-theme` using a local `file:` reference vs. the published NPM package.

This change updates the `importer` to check the sibling `./node_modules` first before traversing up to the parent `node_modules`.